test/image-prepare

```diff
@@ -119,6 +119,10 @@ def build_install_rhel8(dist_tar, image, verbose, quick):
     # in distropkg, keep basic OS packages, otherwise build/install cockpit
     if 'distropkg' not in image:
         args += [
+            # remove already installed packages
+            "--run-command",
+            "if rpm -q cockpit-ws >/dev/null 2>&1; then rpm --erase --nodeps --verbose cockpit cockpit-ws cockpit-bridge cockpit-system; fi",
```

Comment:
Please move this commit to the first in the series, so that logically each commit can pass integration tests (for bisecting)